community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -484,10 +484,6 @@ public void assertOpen()
         {
             throw new TransactionTerminatedException( reason );
         }
-        if ( closed )
```

Comment:
This change was made for the `MandatoryTransactionsForIndexHitsFacadeTest` failure. The corresponding `KernelStatement::assertOpen` does this test or really it throws when the resource-counting doesn't add up. I thought it made sense to have `KernelTransactionImplementation#assertOpen` also throws if it is closed (`!open`) :)